source/_integrations/onewire.markdown

```diff
@@ -97,6 +97,8 @@ Notes:
 
 The 1-Wire bus can be connected with a remote 1-wire host over a network connection using owfs and owserver.
 
+This can be done with a dedicated external device or via an add-on, such as https://github.com/lrybak/hassio-owserver
```

Comment:
We don't recommend or promote third-party extensions/cards or add-ons in or documentation. As the Home Assistant project doesn't support them nor want to endorse the use of third-party extensions.

../Frenck